File: 2737-row-with-maximum-ones/row-with-maximum-ones.java
class Solution {
    public int[] rowAndMaximumOnes(int[][] mat) {
        int ind=0,mc=0;
       for(int i=0;i<mat.length;i++){
        int c=0;
        for(int j=0;j<mat[0].length;j++){
            if(mat[i][j]==1) c++;
        }
        if(mc<c){
            mc=c;
            ind=i;
        }
        
       } 
       return new int[]{ind,mc};
    }
}

File: 2737-row-with-maximum-ones/row-with-maximum-ones.py
class Solution:
    def rowAndMaximumOnes(self, mat: List[List[int]]) -> List[int]:
        maxi_c=-1
        idx=float('inf')
        for i in range(len(mat)):
            count=0
            for j in range(len(mat[0])):
                if mat[i][j]==1:
                    count+=1
            if count>maxi_c:
                maxi_c=count
                idx=i
        return [idx,maxi_c]